webpack.config.js

```diff
@@ -1,5 +1,8 @@
-const BrotliPlugin = require('brotli-webpack-plugin');
```

Comment:
Webpack 5 supports brotli out of the box? Or why remove it?

Reply:
It wasn't even used in the config, it was commented out 👀

Reply:
Oh, I see. That explains the bundle size. Yeah, let's keep it removed. At least we won't have to inflate it before using in tests.